pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -312,7 +312,11 @@ def get_evaluate_epoch_results(self, test_mode):
 
         # clear mem
         self.eval_loop_results = []
-        return results
+
+        return self._add_predictions_to_results(results)
+
+    def _add_predictions_to_results(self, results):
```

Comment:
I'd suggest removing this function if the only place it is called from is 2 lines above

Reply:
Done !